### PR TITLE
Bottom bar

### PR DESCRIPTION
**Pull request by jasonTelanoff**
**Please follow this checklist. Please check each appropriate box (put an 'x' or check it after creating the PR).**
- [x] Provided detailed pull request description and a succinct title (consider template below for guidance).
- [x] Tested your changes, especially after any code review iterations.
- [x] Included any relevant screenshots of UI updates.
- [x] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).
- [x] Verified all contributions are properly licensed pursuant to the [LICENSE](https://github.com/WorldHealthOrganization/app/blob/master/LICENSE) file in the root of the repository.
- [x] Verified your name is in the [content/credits.yaml](https://github.com/WorldHealthOrganization/app/blob/master/content/credits.yaml) file (if you want it to be).

After all boxes above are checked, request and receive an Approved review from any team member knowledgable in the area (TODO team member list).  Once approved, the team member will assign your review to a Committer or use the `needs-merge` label.

## What does this PR accomplish?

Adds a bottom share bar.

Has an opacity less than 255. Share icon is not the default size.

## Did you add any dependencies?

No.

## How did you test the change?

Tested on physical android phone.
Screenshot_20200327-141510
Screenshot_20200327-143307

### CODE REVIEW

**Comment by theswerd** [comment]
This PR has homepage changes from your old branch and will cause a merch conflict

File: client/flutter/lib/home_page.dart [hunk 1]
```diff
@@ -1,49 +1,71 @@
...
-      mainAxisSize: MainAxisSize.max,
-      mainAxisAlignment: MainAxisAlignment.center,
-      crossAxisAlignment: CrossAxisAlignment.stretch,
-      children: <Widget>[
-        Spacer(flex: 1),
-        PageButton(
-            title: S.of(context).protectYourself,
-            onPressed: () => Navigator.of(context)
-                .push(MaterialPageRoute(builder: (c) => ProtectYourself()))),
+        body: StaggeredGridView.count(
```
Comment (theswerd) [inline client/flutter/lib/home_page.dart]: Not for this

Comment (theswerd) [reply client/flutter/lib/home_page.dart]: This branch is bottom bar, and this code is already merged

File: client/flutter/lib/listViewPage.dart [hunk 2]
```diff
@@ -7,15 +8,36 @@ class ListOfItemsPage extends StatelessWidget {
...
           ),
+          Positioned(
+              bottom: 0.0,
+              child: Center(
+                  child: Container(
+                color: Color(0xF8ffffff),
+                height: 60,
+                width: MediaQuery.of(context).size.width,
+                child: Align(
+                  alignment: Alignment.topRight,
```
Comment (theswerd) [inline client/flutter/lib/listViewPage.dart]: should be centerRight

**Comment by jasonTelanoff** [comment]
We're going to try this again in a new PR to avoid merge conflicts.

**Pull request closed without merging**